doc/onboarding.md

```diff
@@ -32,33 +33,38 @@ onboarding session.
 ## Local setup
 
   * git:
-    * make sure you have whitespace=fix: `git config --global --add apply.whitespace fix`
-    * usually PR from your own github fork
+    * Make sure you have whitespace=fix: `git config --global --add apply.whitespace fix`
+    * Always continue to PR from your own github fork
```

Comment:
There is an edge case. If the PR is against the fork's `master` you can't force push a rebased ref set, or if you can you lose the ability to edit.
So maybe:
Continue to always create PRs from __branches__ on your own github fork